.eslintrc.json

```diff
@@ -0,0 +1,20 @@
+{
+    "env": {
+        "browser": true,
+        "es2021": true
+    },
+    "extends": "plugin:react/recommended",
+    "parserOptions": {
+        "ecmaFeatures": {
+            "jsx": true
+        },
+        "ecmaVersion": 12,
+        "sourceType": "module"
+    },
+    "plugins": [
+        "react"
+    ],
+    "rules": {
+        "react/prop-types": "off"
```

Comment:
Why are we disabling prop-types here?

Reply:
earlier I was getting some error related to prop types. Could you please verify now.

Reply:
Disabling prop types from all components is not a good practice and we may have some unwanted bugs/issues in the application because of this. IMHO veerji, it should be removed. Let me know if you need help in sorting out things of prop types.

Reply:
eh tan fix krta veer ji. Can you check the updated code.